packages/core/src/tracing/index.js

```diff
@@ -23,8 +23,6 @@ let automaticTracingEnabled = false;
 let cls = null;
 /** @type {import('../util/normalizeConfig').InstanaConfig} */
 let config = null;
-/** @type {Array.<string>} */
-let extraHeaders = [];
```

Comment:
We kept a copy of the extraHeaders setting in this module but never used it for anything, so I removed it.